connector/connect/common/src/main/scala/org/apache/spark/sql/connect/client/CustomSparkConnectBlockingStub.scala

```diff
@@ -61,9 +61,8 @@ private[connect] class CustomSparkConnectBlockingStub(
         request.getSessionId,
         request.getUserContext,
         request.getClientType,
-        // Don't use retryHandler - own retry handling is inside.
```

Comment:
Change this comment to 
`// retryHandler is used inside ExecutePlanResponseReattachableIterator, don't wrap it here`
I think it's still relevant

Reply:
ah, got it, let me keep the comment.

Reply:
Removed some returns :)